misc/python/materialize/version_list.py

```diff
@@ -258,6 +258,7 @@ def get_published_minor_mz_versions(
     newest_first: bool = True,
     limit: int | None = None,
     include_filter: Callable[[MzVersion], bool] | None = None,
+    exclude_current_minor: bool = True,
```

Comment:
I am not that happy with this rather unexpected default. Can we explicitly set it to `True` in upgrade checks?